crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -1545,6 +1546,10 @@ impl OlmMachine {
         tracing::Span::current()
             .record("sender", debug(&event.sender))
             .record("event_id", debug(&event.event_id))
+            .record(
+                "origin_server_ts",
+                timestamp_to_iso8601(event.origin_server_ts).unwrap_or("<out of range>".to_owned()),
```

Comment:
sorry, this might break up formatting, but using `unwrap_or_else()` is better in this case: it receives a function parameter that's only executed if the option's value is None, vs in the current code, the `to_owned()` (= memory allocation) would happen even if the option's value is None:

```suggestion
                timestamp_to_iso8601(event.origin_server_ts).unwrap_or_else(|| "<out of range>".to_owned()),
```